ports/espressif/boards/espressif_esp32s3_devkitc_1_n8r2_ros/jtagcfg.cfg

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD configuration file for ESP32-S3 connected via ESP USB Bridge board
+#
+# For example, OpenOCD can be started for ESP32-S3 debugging on
+#
+#   openocd -f board/esp32s3-bridge.cfg
+#
+
+# Source the ESP32-S3 configuration file
+source [find target/esp32s3.cfg]
+adapter speed 1000
```

Comment:
I think this file was checked in by accident.

Reply:
It is GPL licensed, so I didn't want to inject that license if not necessary.

Reply:
understandable, I'll remove it